src/components/BadgeComponent.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+  <span class="badge" :class="'badge-' + type">
+    <slot></slot>
+  </span>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from "vue-property-decorator";
+
+@Component({
+  props: {
+    type: String,
+    btnNotification: Boolean
```

Comment:
Please change btnNotification for type with the following options ['regular', 'button', 'link'] (regular is default if is missing)

Reply:
I didn't get the idea, do you mean add those properties to the props object and use those instead of the btnNotification?